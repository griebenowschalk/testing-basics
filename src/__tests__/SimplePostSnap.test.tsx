import { SimplePostSnap } from "../components/Snapshots/SimplePostSnap";
import { render } from "@testing-library/react";

describe("SimplePostSnap snapshot test suite", () => {
  it("initial test", () => {
    //If the snapshot is not updated, the test will fail when the component is updated
    // Normally used when making a big refactor on a component
    const rendered = render(<SimplePostSnap content="Hello" user="John" />);
    expect(rendered.asFragment()).toMatchSnapshot();
  });
});
