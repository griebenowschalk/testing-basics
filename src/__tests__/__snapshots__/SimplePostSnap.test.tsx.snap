// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SimplePostSnap snapshot test suite > initial test 1`] = `
<DocumentFragment>
  <div
    data-testid="post-container"
  >
    <h2>
      John:
    </h2>
    <p>
      Hello
    </p>
  </div>
</DocumentFragment>
`;
